oat-sa/extension-tao-itemqti
# Fix/aut 937/backport

**Ticket:** https://oat-sa.atlassian.net/browse/AUT-937

**How to test:**
- Create an item
- Add any type of interaction to the item
- Select any of predefined font-family values
- Save the item
- Check that the correct font is selected
- Open item again
- Check that the correct font is selected
- Export the item
- Check that styles file is present
